src/tvu.rs

```diff
@@ -351,4 +305,40 @@ mod tests {
         t_l_gossip.join().expect("join");
         t_l_listen.join().expect("join");
     }
+    pub struct TestNode {
+        pub data: ReplicatedData,
+        pub gossip: UdpSocket,
```

Comment:
Can you either add the suffix `_socket` or put all the sockets into an inner struct and name the field `sockets`?